appveyor.yml

```diff
@@ -14,25 +14,24 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
         throw "There are newer queued builds for this pull request, failing early." }
 
-  - cmd: rmdir C:\cygwin /s /q
-
   # Add path, activate `conda` and update conda.
   - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
+  - cmd: conda config --set always_yes yes --set changeps1 no --set show_channel_urls true --set auto_update_conda false
+  - cmd: conda update conda
+  # We need to pin conda until https://github.com/conda/conda/issues/6556 is fixed.
+  - cmd: conda config --system --add pinned_packages defaults::conda
```

Comment:
@czender I update some other configuration options in this PR but this is the line that fixes the issue.